src/inc/msquic.h

```diff
@@ -1482,6 +1495,7 @@ typedef struct QUIC_API_TABLE {
     QUIC_CONNECTION_SET_CONFIGURATION_FN
                                         ConnectionSetConfiguration;
     QUIC_CONNECTION_SEND_RESUMPTION_FN  ConnectionSendResumptionTicket;
+    QUIC_CONNECTION_COMP_RESUMPTION_FN  ConnectionResumptionTicketValidationComplete;
```

Comment:
In order to not make this a breaking change, we need to put it at the end of the struct.